bobtemplates/plone/theme_barceloneta/DEVELOP_THEME.rst.bob

```diff
@@ -0,0 +1,42 @@
+Theme development
+-----------------
+
+In order to develop the theme you need to install grunt-cli global::
+
+    $ sudo npm install -g grunt-cli
+
+You can use the helper script `npm-install`::
+
+    $ ./bin/npm-install
+
+It will run npm install in package root and in the theme folder.
+
+After executing these commands you can run grunt on the package root folder to watch for any less changes::
+
+    $ grunt watch
+
+This will make sure that the many .less files are compiled to .css on the fly and then served up from the theme.
+
+You can also use this command to automaticly reload the browser after changes::
+
+    $ grunt plone-bsync
+
+It will run a Grunt browser sync task in a new browser window and will reload after every change of the less files in the less folder. If your Plone is not running on port 8080 you have to adjust the proxy option in the Gruntfile.js.
+
+If you prefer to do a one time compile of the less files you can run::
+
+    $ grunt compile
+
+or use the helper script from buildout::
+
+    $ ./bin/grunt-task-compile
+
+Actually on a buildout run buildout will run the npm-install and the grunt-task-compile script for you. So you only need the commands above while developing gin the filesystem.
```

Comment:
same here, alternatively we could skip it but document how it can be archived manually or leave it out at all.

Reply:
You are right.

I have removed the references to the buildout-way and explained how to do it using regular npm install.

Thanks,